src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -57,5 +58,103 @@ public static void Close()
 
         private static bool Attach() => AndroidJNI.AttachCurrentThread() == 0;
         private static AndroidJavaObject GetSentryJava() => new AndroidJavaClass("io.sentry.Sentry");
+
+        public static void WriteScope(
+            int? GpuId,
+            string? GpuName,
+            string? GpuVendorName,
+            int? GpuMemorySize,
+            string? GpuNpotSupport,
+            string? GpuVersion,
+            string? GpuApiType,
+            int? GpuMaxTextureSize,
+            bool? GpuSupportsDrawCallInstancing,
+            bool? GpuSupportsRayTracing,
+            bool? GpuSupportsComputeShaders,
+            bool? GpuSupportsGeometryShaders,
+            string? GpuVendorId,
+            bool? GpuMultiThreadedRendering,
+            string? GpuGraphicsShaderLevel)
+        {
+            if (Attach())
+            {
+                using var gpu = new AndroidJavaObject("io.sentry.protocol.Gpu");
```

Comment:
If the type isnt' found, is this going to blow up? (proguard)

Reply:
yes, see #797